.circleci/config.yml

```diff
@@ -108,6 +108,11 @@ commands:
           name: Run database migrations
           command: |
             cf run-task << parameters.app_name >> "yarn db:migrate:prod"
+      - run:
+          name: Reseed database
+          command: |
+            cf run-task << parameters.app_name >> "yarn db:seed:undo"
+            cf run-task << parameters.app_name >> "yarn db:seed:all"
```

Comment:
Shouldn't this be just 'yarn db:seed'?

Reply:
yep, one of the reasons the latest push failed to deploy ha...